packages/office-ui-fabric-react/src/components/FocusZone/FocusZone.tsx

```diff
@@ -349,6 +349,26 @@ export class FocusZone extends BaseComponent<IFocusZoneProps, {}> implements IFo
           }
           return;
 
+        case KeyCodes.tab:
+          if (this.props.allowTabKey) {
```

Comment:
if allowTabKey is true the user will no longer be able to move out of a focusZone (unless it is being handled outside of the FocusZone), correct? That should be noted on the prop so developers know the implications of setting this prop to true.